src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java

```diff
@@ -91,9 +92,18 @@
   @GetMapping(value = "/api/public/courses", produces = "application/json")
   public ResponseEntity<String> getCourses() throws JsonProcessingException {
     List<Course> courseList = courseRepository.findAll();
+
+    String currentQuarter = "F20"; //This should be deleted once configuration file is implemented
+
+    List<Course> currentCourses = new ArrayList<Course>();
+    for(Course i: courseList){
+      if(i.getQuarter().equalsIgnoreCase(currentQuarter)){
```

Comment:
On Codecov it looks like line 100 is yellow, so might need to write a test that makes the if statement true